public/sass/partials/_best-way.scss

```diff
@@ -0,0 +1,107 @@
+.best-way {
+	h1 {
+		font: $head-largest-bold-font;
+		color: $color-3;
+	}
+	h2 {
+		font: $head-big-medium-font;
+		text-align: center;
+		margin-top: 4em;
+		position: relative;
+	}
+	img {
+		display: none;
+		max-width: 13em;
+		@media screen and (min-width: 550px) {
+			max-width: 25em;
+			display: initial;
+		}
+	}
+	.title-text {
+		h1{
+			z-index: -1;
+		}
+		p {
+			position: relative;
+			z-index: -1;
+		}
+		width: 34em;
+		padding: 1em;
+		@media screen and (min-width: 550px) {
+			padding: 2em;
+		}
+	}
+	.title-wrapper {
+		@include flexbox;
+		align-items: center;
+		margin-top: 1em;
+		@media screen and (min-width: 550px) {
+			margin-top: 2em;
+		}
+		@media screen and (min-width: 1030px) {
+			margin-top: 6em;
+		}
+		p {
+			color: $color-3;
+			font: $head-small-light-font;
+			span {
+				font: $head-small-bold-font;
+			}
+		}
+	}
+	.highlight-link {
+		width: 10em;
+		height: 3.5em;
+		margin: 2.5em 6em 2em 0;
+		padding: .5em 0;
+		background-color: $color-2;
+		color: $color-3;
+		border-color: $color-7;
+		font: $small-button-link-font;
+		&:hover {
+			border-color: $color-2;
+			background-color: $color-7;
+		}
+	}
+	.get-it-free {
```

Comment:
there is a lot of code here nested under this parent selector. I would argue that it's not necessary at all to have this name spaced like this.